backend/src/test/java/heartbeat/service/source/github/GithubServiceTest.java

```diff
@@ -180,6 +189,28 @@ void shouldReturnNonRedundantGithubReposWhenCallGithubFeignClientApi() {
 	}
 
 	@Test
+	void shouldReturnGithubTokenIsVerifyWhenVerifyToken() {
+		String githubToken = "123456";
```

Comment:
`githubToken` should use `GITHUB_TOKEN` in the testFixtures.java

same as others...